ContriHUB/ContriHUB-24
# Merge pull request for Issue#166 from ContriHUB/main level color display

Title
Level Color Display Enhancement in Issue Model

Description
In this PR, I have updated the Issue model in the Django application to include a method for displaying colors based on the difficulty level of issues. The changes improve the user experience by allowing the frontend to dynamically color-code issues based on their difficulty, making it easier for users to prioritize their tasks visually.

Summary of Changes:
Updated the get_levelcolor_display method in the Issue model to ensure consistency with the difficulty choice values.
Ensured that color mappings correspond to stored values for accurate representation.
Related Issue:
This enhancement addresses issue #XX (replace with the actual issue number).

Type of Change
 New feature (non-breaking change which adds functionality)
Motivation and Context
This change is required to provide visual differentiation for issues based on their difficulty level, helping users quickly identify which tasks may require more effort. This improvement enhances the usability of the application.

How Has This Been Tested?
The changes were tested in the following manner:

Manual Testing:

Created multiple Issue instances with different difficulty levels.
Rendered the issues in the template and confirmed that the color displayed matched the expected values.
Testing Environment:

Django version: 3.x (replace with your version)
Python version: 3.x (replace with your version)
Browsers: Tested in Chrome and Firefox for UI consistency.
Screenshots (if appropriate):
(Include screenshots of the rendered issues in the UI, showing the correct color for each difficulty level.)

Checklist:
 I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).
 My code follows the code style of this project.
 I have commented my code, particularly in hard-to-understand areas.
 My changes generate no new warnings.
 My change requires a change to the documentation.
 I have updated the documentation accordingly.
 I have read the CONTRIBUTING document.
 Any dependent changes have been merged and published in downstream modules.
 I have added tests to cover my changes.
 All new and existing tests passed.
Test Configuration:
Firmware version: Not applicable
Hardware: Development machine (specify OS if relevant, e.g., Windows 10, Ubuntu 20.04)
Toolchain: Django, Python
SDK: Not applicable
